Examples/RPCWebSocketTest.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<HTML>
+<head>
+  <meta charset="utf-8" />
+  <title>SEGS RPC / WebSocket Test</title>
+
+<style>
+.button {
+    background-color: #4CAF50; /* Green */
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+}
+
+.service_button {
+  background-color: #grey
+  border: none;
+  color: red;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  width: 150px;
+  height: 40px;
+}
+
+</style>
+
+<body>
```

Comment:
Pretty unconventional to put `<body>` inside `<head>`, normally this section would be below `</head>`, hence the names. :)